File: ETicaretProje.Backend/EticaretProje.Domain/Concrete/CartProduct.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using EticaretProje.Domain.Abstract;

namespace EticaretProje.Domain.Entities
{
    public class CartProduct : BaseObject
    {
        public int ProductId { get; set; }
        public string ProductName { get; set; }
        public float Price { get; set; }
        public int? CartId { get; set; }
        public int? OrderId { get; set; }
        public int Quantity { get; set; }
    }
}

File: ETicaretProje.FrontEnd/src/app/models/Classes/CartProduct.ts
export class CartProduct {
  productId: number;
  productName:string;
  price:number;
  cartId: number;
  quantity: number;
  isActive?: boolean;
  /**
   *
   */
  constructor(productId: number,productName:string,price:number,cartId: number,quantity: number) {
    this.productId = productId;
    this.cartId = cartId;
    this.quantity = quantity;
    this.productName = productName;
    this.price = price;
    this.isActive = true;
  }
}